### ISSUE TITLE
Touchpoints modal in app doesn't maintain focus

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#4195]
## Area or Component
in app - any page

## Quick Summary
The modal for Touchpoints doesn't hold focus, so screenreaders can read outside it and users can tab outside the modal.

See this same issue for the static site here: https://github.com/CDCgov/prime-simplereport/issues/4030

### How to Replicate
1. Open Touchpoints in the SR app
2. Start tabbing backwards
3. See that you go back to the main page content

## Accessibility Violation
Able to browse outside modal with screen reader: https://cdc-devtools.dequecloud.com/rule-help/en/reading-order-browse-outside-modal
Keyboard focus is not maintained in modal: https://cdc-devtools.dequecloud.com/rule-help/en/focus-modal-moves-outside

## Action Requested
Ensure that keyboard focus can't escape the Touchpoints modal.

**Comment by emyl3** [comment]
Moving this back to the backlog but we may want to icebox/close this depending on the designs Kenny comes up with. For more info: https://github.com/CDCgov/prime-simplereport/pull/4310
